hack/install-kubectl.sh

```diff
@@ -5,33 +5,32 @@
 # SPDX-License-Identifier: Apache-2.0
 
 #
-# Installs "kubectl" on Travis-CI Ubuntu.
+# Installs "kubectl"
 #
 
-set -eu
+set -euo pipefail
 
-sudo apt-get update > /dev/null && \
-    sudo apt-get install -y \
-        apt-transport-https \
-        curl \
-        git
+# Find a suitable install location
+if [[ -w /usr/local/bin ]]; then
```

Comment:
I would rather see `${HOME}/bin` as the first location to be checked since this is not an official `installation script`.

Reply:
@kevydotvinu I think it is a good point that @coreydaley brought up here. Maybe you can change this piece here and the one in https://github.com/shipwright-io/build/pull/819 to this one:
```sh
# Find a suitable install location
for CANDIDATE in "$HOME/bin" "/usr/local/bin" "/usr/bin"; do
  if [[ -w $CANDIDATE ]] && grep -q "$CANDIDATE" <<<"$PATH"; then
    TARGET_DIR="$CANDIDATE"
    break
  fi
done

# Bail out in case no suitable location could be found
if [[ -z ${TARGET_DIR:-} ]]; then
  echo -e "Unable to determine a writable install location. Make sure that you have write access to either \\033[1m/usr/local/bin\\033[0m or \\033[1m${HOME}/bin\\033[0m and that is in your PATH."
  exit 1
fi
```

I tried it out in my repository and this should be better to read and better to maintain in terms of the order.

Reply:
I have made the suggested changes in both places. Thanks, @HeavyWombat @coreydaley.